### PR TITLE
New notification type: alert-manually-deleted

### PR DESCRIPTION
**Pull request by cuchi**
@haskellcamargo can you help me with the `fr` & `es` translations?

### CODE REVIEW

**Review by williamrozin — APPROVED** [review]

**Review by larissathasdefar — APPROVED** [review]

File: src/screens/NotificationList.re [hunk 1]
```diff
@@ -79,17 +81,20 @@ let getNotificationStyles = (notification) =>
...
+        |> replace("{{COUNT}}", string_of_int(countAlerts(notification)))
+        |> replace("{{APP}}", notification##name))
     | "permissions-updated" => ("security", "pink", t("permissionsUpdated"))
     | "task-created" => ("warning", "red", t("taskCreated"))
     | "alert-comment" => ("comment", "indigo", t("alertComment"))
     | "alert-follow" => ("people", "brown", t("alertFollow"))
     | "alert-unfollow" => ("do_not_disturb", "red", t("alertUnfollow"))
+    | "alert-manually-deleted" => ("delete_forever", "red",
+        t("alertDeleted")
+        |> replace("{{TTILE}}", notification##name))
```
Comment (haskellcamargo) [inline src/screens/NotificationList.re]: `TTILE`?

**Review by haskellcamargo — CHANGES_REQUESTED** [review]

File: src/screens/NotificationList.re (hunk 1 above)
Comment (cuchi) [reply src/screens/NotificationList.re]: ops

File: _locales/en/messages.json [hunk 2]
```diff
@@ -74,5 +74,9 @@
     "alertUnfollow": {
         "message": "An alert got new unfollows",
         "description": "Alert unfollow"
+    },
+    "alertDeleted": {
+        "message": "The alert {{TITLE}} was deleted",
+        "description": "Alert deleted"
```
Comment (haskellcamargo) [inline _locales/en/messages.json:80]: ## French:
- Le alerte {{TITLE}} a été exclu
- Alerte exclu
## Spanish:
- Lo alerta {{TITLE}} ha sido borrado
- Alerta borrado

Ah, also, if you can, replace `Um alerta` in `es` by `Un alerta`. My typo.

Comment (cuchi) [reply _locales/en/messages.json:80]: Thanks!

**Review by haskellcamargo — APPROVED** [review]

**Pull request merged**